bentoctl/cli/__init__.py

```diff
@@ -26,6 +28,12 @@
     terraform_destroy,
 )
 
+try:
+    from bentoml._internal.utils.docker import validate_tag
+except ImportError:
+    from bentoctl.cli.utils import dummpy_validate_tag as validate_tag
```

Comment:
how about dropping validation and mapping it to None, for now, would that work? The plan was to release a new version of bentoctl, including this PR. but if you feel it would be better to ignore this PR for the release we can do that too, what do you think?